cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -179,58 +190,116 @@ def _resolve_target(self, target: Union[str, None]) -> str:
 
     def get_counts(
         self,
-        circuit: cirq.Circuit,
+        circuits: Union[cirq.Circuit, Sequence[cirq.Circuit]],
         repetitions: int,
         target: Optional[str] = None,
         param_resolver: cirq.ParamResolverOrSimilarType = cirq.ParamResolver({}),
+        index: Optional[int] = None,
```

Comment:
Sure that makes sense, though is there not the use case of wanting to see the counts of, say, the 2nd circuit from a batch job specifically and offering the index param as a way to avoid the clumsier `get_counts()[1]`?

Reply:
i guess i don't see why i'd run a batch job if i was going to throw away the results from all but one circuit? instead i'd just use e.g. `service.get_counts(my_circuits[2])`

Reply:
That's fair, I guess I wasn't sure about the use case of `get_counts()` initially. Updating now.